mite/runner.py

```diff
@@ -204,8 +204,7 @@ async def _execute(self, context, scenario_id, scenario_data_id, journey_spec, a
                 else:
                     await journey(context, *args)
         except Exception as e:
-            if not getattr(e, "handled", False):
-                if self._debug:
-                    breakpoint()
-                    raise
+            if not getattr(e, "handled", False) and self._debug:
```

Comment:
you could reorder: `if self._debug and getattr(...)`.  this has the advantage of not needing to make a getattr call if debug is false.